tgctoolbox/settings.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import os
+
+import yaml
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+BASE_FILES = [
+    os.path.join("configuration", "base.yaml"),
+    os.path.join("configuration", "local.yaml"),
+    os.path.join("configuration", "production.yaml"),
+]
+

```

Comment:
at the end of the day I would expect there to be a constant value `SETTINGS` holding the instance of the settings, so that it was accessible everywhere.

At least that's how i see the things should be done.

Reply:
There is one instance of settings currently accessible via calling constructor of the class. That's what `__new__` does. All you have to do to access the settings is import Settings class and initialize it. Those BASE_Files will always be used anyways, but I left an option of defining other conf files.